frontend/frontend/private.py

```diff
@@ -189,7 +192,10 @@ async def _process_form(
         # Not set because rj.zadd doesn't require it
         # don't touch! rj.jsonset(f"alg-{name}-queries", root, [])
         try:
-            httpx.post(f"http://backend:8400/init/{name}")
+            logger.info("initializing algorithm {name}...")
+            r = httpx.post(f"http://backend:8400/init/{name}")
+            assert r.status_code == 200
```

Comment:
Add error message pr raise an `HTTPException`? I don't know what this try/except is catching.